javascript/forevervm/src/run.js

```diff
@@ -6,7 +6,11 @@ import { getBinary } from './get-binary.js'
 async function runBinary() {
   let binpath = await getBinary()
 
-  spawnSync(binpath, process.argv.slice(2), { stdio: 'inherit', stderr: 'inherit' })
+  spawnSync(binpath, process.argv.slice(2), {
+    stdio: 'inherit',
+    stderr: 'inherit',
+    env: { ...process.env, FOREVERVM_RUNNER: 'npx', FOREVERVM_SDK: 'javascript' },
```

Comment:
I think `FOREVERVM_SDK` shouldn't be set here, right? We're not actually using the forevervm JS SDK in this code path

Reply:
whoops missed this one, good catch!